src/test/librbd/test_mock_fixture.h

```diff
@@ -25,7 +25,11 @@ ACTION_P(CopyInBufferlist, str) {
 }
 
 ACTION_P2(CompleteContext, r, wq) {
+#if defined(__clang__)
+  ContextWQ *context_wq = static_cast<ContextWQ *>(wq);
```

Comment:
@wjwithagen please note, i was indeed checking the LLVM's libc++ instead of the GNU's libstdc++ . maybe you could check your `stddef.h` to see how `NULL` is defined?